interfaces/Portalicious/src/app/pages/change-password/change-password.page.html

```diff
@@ -3,49 +3,6 @@
   i18n-pageTitle="@@page-title-change-password"
 >
   <p-card styleClass="max-w-xl">
-    <app-form-default
-      [formGroup]="formGroup"
-      submitButtonText="Change password"
-      i18n-submitButtonText
-      [mutation]="changePasswordMutation"
-    >
-      <app-form-field-wrapper
-        label="Current Password"
-        i18n-label
-        [errorMessage]="formFieldErrors()('currentPassword')"
-      >
-        <p-password
-          formControlName="currentPassword"
-          autocomplete="current-password"
-          [toggleMask]="true"
-          [feedback]="false"
-          pAutoFocus
-          autofocus
-        />
-      </app-form-field-wrapper>
-      <app-form-field-wrapper
-        label="New Password"
-        i18n-label
-        [errorMessage]="formFieldErrors()('newPassword')"
-      >
-        <p-password
-          formControlName="newPassword"
-          autocomplete="new-password"
-          [toggleMask]="true"
-          [feedback]="false"
-        />
-      </app-form-field-wrapper>
-      <app-form-field-wrapper
-        label="Confirm Password"
-        i18n-label
-        [errorMessage]="formFieldErrors()('confirmPassword')"
-      >
-        <p-password
-          formControlName="confirmPassword"
-          [toggleMask]="true"
-          [feedback]="false"
-        />
-      </app-form-field-wrapper>
-    </app-form-default>
+    <ng-container *ngComponentOutlet="ChangePasswordComponent"></ng-container>
```

Comment:
This way, maybe some day, we could return this page back to its "User settings"-purpose.. ;)